install/kubernetes/cilium/values.yaml

```diff
@@ -480,3 +480,11 @@ global:
   ci:
     # Make Cilium panic if objects received by k8s are modified.
     kubeCacheMutationDetector: false
+
+  ipam:
+    mode: "operator"
+    operator:
+      ipv4PodCIDR: "10.0.0.0/16"
```

Comment:
This is too small. Only supporting 255 nodes by default is very limiting. I suggest we default to `10.0.0.0/8`.